### PR TITLE
Add website record example

### PR DESCRIPTION
**Pull request by reinismu**
This is how I wrote it. It might not be the best way. Let me know if there is anything to improve in my pipeline :)

### CODE REVIEW

File: gstcefdemux.cc [hunk 1]
```diff
@@ -4,7 +4,7 @@
 #include "gstcefaudiometa.h"
 
 #define CEF_VIDEO_CAPS "video/x-raw, format=BGRA, width=[1, 2147483647], height=[1, 2147483647], framerate=[1/1, 60/1]"
-#define CEF_AUDIO_CAPS "audio/x-raw, format=F32BE, rate=[1, 2147483647], channels=[1, 2147483647], layout=interleaved"
+#define CEF_AUDIO_CAPS "audio/x-raw, format=F32LE, rate=[1, 2147483647], channels=[1, 2147483647], layout=interleaved"
```
Comment (MathieuDuponchelle) [inline gstcefdemux.cc:7]: why? :)

File: README.md [hunk 2]
```diff
@@ -18,3 +18,17 @@ GST_PLUGIN_PATH=Release:$GST_PLUGIN_PATH gst-launch-1.0 \
     video/x-raw, width=1920, height=1080, framerate=60/1 ! cefdemux name=d d.video ! queue ! videoconvert ! \
     xvimagesink d.audio ! queue ! audioconvert ! pulsesink
 ```
+
+Record website video + audio (with audiomixer)
```
Comment (MathieuDuponchelle) [inline README.md:22]: can you also add an audiomixer in the playback pipeline? :)

File: README.md [hunk 3]
```diff
@@ -18,3 +18,17 @@ GST_PLUGIN_PATH=Release:$GST_PLUGIN_PATH gst-launch-1.0 \
...
+
+``` shell
+gst-launch-1.0 cefsrc url="https://soundcloud.com/platform/sama" ! \
+    video/x-raw, width=1920, height=1080, framerate=60/1 ! \
+    cefdemux name=demux ! videoconvert ! queue max-size-bytes=0 max-size-buffers=0 max-size-time=3000000000 ! x264enc ! \
```
Comment (MathieuDuponchelle) [inline README.md:27]: perhaps reindent so this still looks tidy

File: gstcefdemux.cc (hunk 1 above)
Comment (reinismu) [reply gstcefdemux.cc:7]: Ohh no, I wanted to make it a separate thing. I will remove it

File: README.md [hunk 4]
```diff
@@ -18,3 +18,17 @@ GST_PLUGIN_PATH=Release:$GST_PLUGIN_PATH gst-launch-1.0 \
...
+    mp4mux name=muxer fragment-duration=1000 ! filesink location='test.mp4' \
+    audiotestsrc do-timestamp=true is-live=true  volume=0.00 ! audiomixer name=mix ! \
+    queue max-size-bytes=0 max-size-buffers=0 max-size-time=3000000000 ! audioconvert ! audiorate ! audioresample ! faac bitrate=128000 ! muxer. \
```
Comment (MathieuDuponchelle) [inline README.md:30]: same here

File: README.md (hunk 3 above)
Comment (reinismu) [reply README.md:27]: I have tried to split by concerns at least from how I see it

File: README.md (hunk 2 above)
Comment (reinismu) [reply README.md:22]: I'm not sure I get what you mean. 
In this one?
```
GST_PLUGIN_PATH=Release:$GST_PLUGIN_PATH gst-launch-1.0 \
    cefsrc url="https://soundcloud.com/platform/sama" ! \
    video/x-raw, width=1920, height=1080, framerate=60/1 ! cefdemux name=d d.video ! queue ! videoconvert ! \
    xvimagesink d.audio ! queue ! audioconvert ! pulsesink
```

File: README.md (hunk 3 above)
Comment (reinismu) [reply README.md]: But ye it does end up too long. Split it in more lines

File: README.md (hunk 2 above)
Comment (MathieuDuponchelle) [reply README.md:22]: yes

Comment (reinismu) [reply README.md:22]: Hmm interesting. I can't get it to work :/

```
GST_PLUGIN_PATH=Release:$GST_PLUGIN_PATH gst-launch-1.0 \
    cefsrc url="https://soundcloud.com/platform/sama" ! \
    video/x-raw, width=1920, height=1080, framerate=60/1 ! cefdemux name=d d.video ! queue ! videoconvert ! \
    xvimagesink audiotestsrc do-timestamp=true is-live=true  volume=0.01 ! audiomixer name=mix ! \
    queue ! audioconvert ! pulsesink \
    d.audio ! mix.
```
I can hear `audiotestsrc`, but can't `d.audio`

Comment (MathieuDuponchelle) [reply README.md:22]: hrm that's weird indeed, resolving this then but please open an issue, something's fishy :(

**Comment by MathieuDuponchelle** [comment]
ah hrm, please fix up your two commits :)

File: README.md (hunk 2 above)
Comment (MathieuDuponchelle) [reply README.md:22]: actually, also works fine with larger queues:

```
gst-launch-1.0     cefsrc url="https://soundcloud.com/platform/sama" !     video/x-raw, width=1920, height=1080, framerate=60/1 ! cefdemux name=d d.video ! queue max-size-bytes=0 max-size-buffers=0 max-size-time=3000000000 ! videoconvert !     xvimagesink d.audio ! queue ! audioconvert ! queue max-size-bytes=0 max-size-buffers=0 max-size-time=3000000000 ! audiomixer name=mix ! autoaudiosink audiotestsrc is-live=true volume=0.0 ! queue ! mix.
```

Comment (reinismu) [reply README.md:25]: Ou I see I had to increase both queues. I tried before by just increasing one of them. Interesting.
Updated readme.md

Comment (reinismu) [reply README.md:25]: Actually. Do we increase queues? I checked `gst-inspect-1.0 queue` and 
```
  max-size-buffers    : Max. number of buffers in the queue (0=disable)
                        flags: readable, writable, changeable in NULL, READY, PAUSED or PLAYING state
                        Unsigned Integer. Range: 0 - 4294967295 Default: 200 
  max-size-bytes      : Max. amount of data in the queue (bytes, 0=disable)
                        flags: readable, writable, changeable in NULL, READY, PAUSED or PLAYING state
                        Unsigned Integer. Range: 0 - 4294967295 Default: 10485760 
  max-size-time       : Max. amount of data in the queue (in ns, 0=disable)
                        flags: readable, writable, changeable in NULL, READY, PAUSED or PLAYING state
                        Unsigned Integer64. Range: 0 - 18446744073709551615 Default: 1000000000 
```
From what I see we decrease them :D

Comment (MathieuDuponchelle) [reply README.md:25]: no we increase them to 3 seconds or an unlimited number of bytes / buffers

Comment (reinismu) [reply README.md:25]: Ahh I see was confused with `(0=disable)` thought it removes buffers. `-1` is wording what I'm used to

**Comment by MathieuDuponchelle** [comment]
Thanks!

**Pull request merged**